src/main/common/interest.ts

```diff
@@ -15,11 +15,11 @@ export async function getInterestDetails (claim: Claim): Promise<object> {
   if (claim.claimData.interestDate.type === InterestDateType.CUSTOM) {
     interestDate = claim.claimData.interestDate.date
   } else {
-    interestDate = claim.createdAt
+    interestDate = claim.issuedOn
```

Comment:
Are you aware that issue date can be the next day if claim issued after 4pm?

Change will cause that on the UI we would say `Interest from 14th Feb to 13th Feb`.

Reply:
Maybe the solution is just to strip time out of creation date. This way we would align date calculation with interest calculation as calculation in claim service does not take time in consideration when interest is calculated.

Reply:
That is why I have switched to using the issue date as it will take into account that time difference

Reply:
Calculating interest from issue date is much different then calculating interest from case submission date. Do you have business agreement for changing the calculation logic? Issue date might be few days after the submission date as issue date must be working day.